core/src/main/java/org/apache/carbondata/core/metadata/SegmentFileStore.java

```diff
@@ -105,7 +105,8 @@ public SegmentFileStore(String tablePath, String segmentFileName) throws IOExcep
    */
   public static void writeSegmentFile(String tablePath, final String taskNo, String location,
```

Comment:
main method name also can be changed i guess, as internally it always called to write the segment file for partition table.

Reply:
done